docs/core-concepts/architecture.md

```diff
@@ -58,9 +58,12 @@ Citrine is set up that Modules have the actual implementations of OCPP Functiona
 They pull in the other packages like 00_base, 01_Data, 02_Util and are built on top.
 
 The structure for each module should be roughly the same, set up as:
-- `api.ts`: Defines the API endpoints for the module
+
+- `DataApi.ts`: Defines API endpoints specific to data-related interactions.
+- `MessageApiOcpp106.ts`: Defines API endpoints specific to OCPP 1.6 messages.
+- `MessageApiOcpp201.ts`: Defines API endpoints specific to OCPP 2.0.1 messages. 
```

Comment:
The file name isn't fully match to the current name `MessageApi.ts`. How about changing to `1.6/MessageApi.ts` and `2.0.1/MessageApi.ts`?